sc-poc/modules/auth/handler_auth_opa.go

```diff
@@ -84,7 +84,7 @@ func (h *AuthOPAHandler) ServeHTTP(w http.ResponseWriter, r *http.Request, next
 		policyName = utils.Hash(h.Rego)
 	}
 
-	allowed, reason, err := h.authApp.EvaluateOPAPolicy(r.Context(), policyName, input)
+	allowed, reason, err := h.authApp.policies[policyName].Evaluate(r.Context(), input)
```

Comment:
Don't do this. Let the handler call a function on the AuthApp. The same way it did before.